devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusTask.java

```diff
@@ -20,6 +21,9 @@
 import io.quarkus.utilities.OS;
 
 public abstract class QuarkusTask extends DefaultTask {
+    private static final String[] WORKER_BUILD_FORK_OPTIONS = { "quarkus.package",
+            "quarkus.application", "quarkus.gradle-worker" };
```

Comment:
Let's add a `.` because you never know if we won't add `quarkus.application-foo` one day.
```suggestion
    private static final String[] WORKER_BUILD_FORK_OPTIONS = { "quarkus.package.",
            "quarkus.application.", "quarkus.gradle-worker." };
```

Reply:
That's a good and reasonable suggestion, I added the trailing `.`